### ISSUE TITLE
V11 update?

### ISSUE THREAD

**Issue by Avitale34** [paulcheeba/weather-reports#10]
is this gonna be updated for v11?

**Comment by paulcheeba** [comment]
Yes, I've just been quite busy at work the last month or so. I also had to confirm with (Foundry mods) that the latest update didn't destroy my work. I'll look into this and a couple other issues next week if I can dedicate the time.

**Comment by Avitale34** [comment]
Just popping back in! Love this module!

**Comment by sirkerry** [comment]
Any news on this front or even updating to v12?

**Comment by paulcheeba** [comment]
Updated to v12, including v11 compatibility. Sorry for the wait.
